src/GoogleFit/FitOnFhir.GoogleFit/Services/GoogleFitImportService.cs

```diff
@@ -87,7 +98,8 @@ public class GoogleFitImportService : ParallelTaskWorker<GoogleFitImportOptions>
                             _logger.LogInformation("Push Dataset: {0}", dataStreamId);
```

Comment:
When you log the event hub batch size and message count, you should include the stream id. Because there will be many operations running in parallel, if you don't add the stream id, it will be more difficult to correlate them in the logs.

Also, this will always result in "size 0 and count 0" because you are creating the new batch and calling the log before anything has been added.

Reply:
This is just logging the datastream ID, not the `MedTechDataset` or anything like that.

Reply:
I mean this line:

```c#
_logger.LogInformation("Created Eventhub Batch (size {0}, count {1})", eventBatch.SizeInBytes, eventBatch.Count);
```